--- ../lib/db/cortex.php
+++ ../lib/db/cortex.php
@@ -690,9 +690,7 @@
 		if ($this->dbsType == 'sql' && !$count) {
 			$m_refl=new \ReflectionObject($this->mapper);
 			$m_ad_prop=$m_refl->getProperty('adhoc');
-			$m_ad_prop->setAccessible(true);
 			$m_refl_adhoc=$m_ad_prop->getValue($this->mapper);
-			$m_ad_prop->setAccessible(false);
 			unset($m_ad_prop,$m_refl);
 		}
 		// sorting by relation helper: @field.field
