--- ../src/DeepCopy/Filter/ReplaceFilter.php
+++ ../src/DeepCopy/Filter/ReplaceFilter.php
@@ -30,9 +30,6 @@
     public function apply($object, $property, $objectCopier)
     {
         $reflectionProperty = ReflectionHelper::getProperty($object, $property);
-        if (PHP_VERSION_ID < 80100) {
-            $reflectionProperty->setAccessible(true);
-        }
 
         $value = call_user_func($this->callback, $reflectionProperty->getValue($object));
 
