--- ../src/DeepCopy/Matcher/PropertyTypeMatcher.php
+++ ../src/DeepCopy/Matcher/PropertyTypeMatcher.php
@@ -39,10 +39,6 @@
             return false;
         }
 
-        if (PHP_VERSION_ID < 80100) {
-            $reflectionProperty->setAccessible(true);
-        }
-
         // Uninitialized properties (for PHP >7.4)
         if (method_exists($reflectionProperty, 'isInitialized') && !$reflectionProperty->isInitialized($object)) {
             // null instanceof $this->propertyType
